src/client/components/not_found/index.tsx

```diff
@@ -6,7 +6,7 @@ export class NotFoundPage extends Component {
     return (
       <section className="ph4 mt3">
         <h1>Page Not Found</h1>
-        <p>TODO put something fun here</p>
+        <img src='/images/flapjack.png' alt='KBase logo' />
```

Comment:
This won't work in a deploy. We need something like `src="{ url_for('static', filename='images/flapjack.png') }'>`